README.md

```diff
@@ -36,10 +36,10 @@ execute:
 ```
 * In the command line, run `quarto render archive/YYYY-MM-workshop-name`. This will regenerate the workshop slides under `docs/archive/YYYY-MM-workshop-name`.
 * Check that:
-	* Running `quarto render` didn't change any files in `docs/` outside of `docs/archive/`.
+	* Running `quarto render` didn't change any files in `docs/` other than `sitemap.xml` and `search.json` outside of `docs/archive/`.
```

Comment:
This should mention `index.html` as the third file allowed to change inside `docs/archive/`, no?

Reply:
Right, yes!